From 21b341c1fab142b340072fe1ac854b4129fa86f0 Mon Sep 17 00:00:00 2001
From: "Anna (navi) Figueiredo Gomes" <navi@vlhl.dev>
Date: Fri, 25 Apr 2025 18:20:43 +0200
Subject: [PATCH] init.d/user: use longer respawn period

some setups could actually take more than a second to fail 3 times "in a row"

Bug: https://bugs.gentoo.org/952108
---
 init.d/user.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init.d/user.in b/init.d/user.in
index 5b52ce3d..ea416e5c 100644
--- a/init.d/user.in
+++ b/init.d/user.in
@@ -20,7 +20,7 @@ notify="fd:3"
 # If it's rapidly failing, usually due to XDG_RUNTIME_DIR being unset,
 # we should just quit fast.
 respawn_max=3
-respawn_period=1
+respawn_period=5
 
 start_pre() {
 	if [ "$user" = "$RC_SVCNAME" ]; then
